### PR TITLE
Fix rgb change tensors2floats 2

### PR DESCRIPTION
**Pull request by lisapaige**

### CODE REVIEW

File: seg/segment/predict.py [hunk 1]
```diff
@@ -177,7 +177,11 @@ def run(
                 for (*xyxy, conf, cls), mask in zip(reversed(det[:, :6]),
                                                     reversed(masks)):
                     xywh = (xyxy2xywh(torch.tensor(xyxy).view(1, 4)) / gn).view(-1).tolist()  # normalized xywh
+                    line = (cls, *xywh, conf) if save_conf else (cls, *xywh)  # label format
                     if save_txt:  # Write to file
+                        xywh = (xyxy2xywh(torch.tensor(xyxy).view(1, 4)) / gn).view(-1).tolist()  # normalized xywh
+                        line = (cls, *xywh, conf) if save_conf else (cls, *xywh)  # label format
+                        labels.append(line)
```
Comment (oadams) [inline seg/segment/predict.py:184]: If this statement is in the conditional then when `save_text` is falsy then it won't get run. But it should still be run, otherwise the labels this function returns will be incorrect.

File: seg/segment/predict.py [hunk 2]
```diff
@@ -177,7 +177,11 @@ def run(
                 for (*xyxy, conf, cls), mask in zip(reversed(det[:, :6]),
                                                     reversed(masks)):
                     xywh = (xyxy2xywh(torch.tensor(xyxy).view(1, 4)) / gn).view(-1).tolist()  # normalized xywh
+                    line = (cls, *xywh, conf) if save_conf else (cls, *xywh)  # label format
```
Comment (oadams) [inline seg/segment/predict.py:180]: Line no longer has the mask in it with this form, so the functionality has changed. It previously was `                    line = (cls, *xywh, mask == 1, conf) if save_conf else (cls, *xywh, mask == 1)  # label format`
